candidate_tags/t-sne.csv

```diff
@@ -0,0 +1,459 @@
+setup
+64
+show
+sound
+send
+frame
+use
+disable
+laptop
+redirect
+website
+unusable
+wordpress
+button
+hang
+developer
+config
+trouble
+pro
+deleting
+suggestion
+respond
+toolbar
+ad
+preference
+outlook
+71
+pop
+67
+input
+box
+random
+windows10
+access
+restore
+malware
+customize
+device
+home
+control
+text
+features
+local
+server
+setting
+safe
+crash
+right
+script
+glitch
+contrast
+insecure
+manager
+https
+catalina
+signin
+download
+cloud
+adobe
+render
+content
+bank
+external
+url
+resize
+apple
+howto
+63
+online
+install
+white
+loading
+symbol
+blank
+hyperlinks
+origin
+disabled
+backup
+history
+word
+pop-
+title
+theme
+printing
+edition
+release
+program
+network
+thunderbird
+manage
+blocker
+automatically
+plus
+speed
+old
+banking
+popup
+select
+safemode
+link
+manually
+certificate
+erase
+close
+feature
+print
+line
+code
+javascript
+warning
+minimize
+application
+android
+drag
+css
+payment
+sign
+opens
+ask
+70
+audio
+startup
+downgrade
+enable
+lock
+command
+folder
+log
+order
+reload
+verify
+photo
+opera
+open
+attachment
+recovery
+screen
+mail
+ie
+paste
+hard
+notification
+launch
+save
+work
+format
+result
+document
+storage
+mac
+tracking
+software
+pdf
+engine
+gmail
+type
+auto
+stop
+drive
+private
+computer
+filter
+addon
+72
+cache
+google
+login
+fill
+configuration
+time
+language
+address
+find
+session
+previous
+task
+email
+different
+post
+windows
+esr
+site
+configure
+10
```

Comment:
Do you think there is any way you can remove tags like "10" and "72"?

Reply:
Actually keep the numbers. TF-IDF also has numbers. It will be interesting to see how both of these methods compare